compiler/tools/fpp-to-cpp/test/component/base/QueuedTestComponentAc.ref.hpp

```diff
@@ -2946,6 +2950,90 @@ class QueuedTestComponentBase :
         const S& s //!< A struct
     );
 
+  private:
+
+    // ----------------------------------------------------------------------
+    // Invocation functions for special output ports
+    // ----------------------------------------------------------------------
+
+    //! Invoke output port cmdRegOut
+    void cmdRegOut_out(
```

Comment:
Perhaps I missed it earlier....but aren't these to be gated by `#if !FW_DIRECT_PORTS` or is this only in the C++.  Logically it is only the C++.